src/pybamm/experiment/step/base_step.py

```diff
@@ -69,6 +76,10 @@ def __init__(
         description=None,
         direction=None,
     ):
+        if direction is not None and direction not in Direction:
```

Comment:
Something like this could fix the failing tests
```suggestion
        if direction is not None and not isinstance(direction, Direction):
```

Reply:
this doesn't work